packages/react-core/src/components/Brand/Brand.tsx

```diff
@@ -1,18 +1,71 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
-
+import styles from '@patternfly/react-styles/css/components/Brand/brand';
+import { setBreakpointCssVars } from '../../helpers';
 export interface BrandProps
   extends React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement> {
-  /** Additional classes added to the Brand. */
+  /** Transforms the Brand into a <picture> element from an <img> element. Container for <source> child elements. */
+  children?: React.ReactNode;
+  /** Additional classes added to the either type of Brand. */
   className?: string;
-  /** Attribute that specifies the URL of the image for the Brand. */
+  /** Attribute that specifies the URL of a <img> Brand. For a <picture> Brand this specifies the fallback <img> URL. */
   src?: string;
-  /** Attribute that specifies the alt text of the image for the Brand. */
+  /** Attribute that specifies the alt text of a <img> Brand. For a <picture> Brand this specifies the fallback <img> alt text. */
   alt: string;
+  /** Widths are various breakpoints for a <picture> Brand. */
```

Comment:
Super nitpick: "Widths at" to match the description for the heights prop

Reply:
Whoops haha